entity-framework/core/providers/sql-server/functions.md

```diff
@@ -204,14 +204,14 @@ string.IsNullOrEmpty(value)                                             | @value
 string.IsNullOrWhiteSpace(value)                                        | @value IS NULL OR @value = N''
 stringValue.CompareTo(strB)                                             | CASE WHEN @stringValue = @strB THEN 0 ... END
 stringValue.Contains(value)                                             | @stringValue LIKE N'%' + @value + N'%'
-stringValue.EndsWith(value)                                             | @stringValue LIKE N'%' + @value
+stringValue.EndsWith(value)                                             | ENDSWITH(@stringValue, @value)
```

Comment:
Is this right? I recently made some changes in this area, I think we use LIKE for constant and parameter patterns, and LEFT/RIGHT for column patterns (is this maybe the Cosmos translation?)

Reply:
Err… maybe. lol, I’ll revert